crates/cli/src/subcommands/init.rs

```diff
@@ -178,6 +178,8 @@ pub async fn exec_init_csharp(args: &ArgMatches) -> anyhow::Result<()> {
         (include_str!("project/csharp/StdbModule._csproj"), "StdbModule.csproj"),
         (include_str!("project/csharp/Lib._cs"), "Lib.cs"),
         (include_str!("project/csharp/_gitignore"), ".gitignore"),
+        // Reuse global.json from modules instead of a separate template to ensure that we stay in sync.
+        (include_str!("../../../../modules/global.json"), "global.json"),
```

Comment:
This is going to cause an issue with publishing the module: https://github.com/rust-lang/cargo/issues/13309

Please revise.

Reply:
We discussed that, but we can't do that because the dry-run fails because we aren't actually pushing things and they depend on each other.

Reply:
Ok not a huge fan of copying, but given that this is going to change _very_ rarely (once a year at worst), I just copied the JSON file into the template directory like all other files.